src/reanimated2/interpolation.js

```diff
@@ -51,3 +52,24 @@ export function interpolate(x, input, output, type) {
   }
   return internalInterpolate.apply({}, [x].concat(narrowedInput).concat(type));
 }
+
+export function interpolateColor(x, input, output, type) {
+  'worklet';
+
+  // Extrapolate.CLAMP is the default interpolation type for colors
+  type = type || Extrapolate.CLAMP;
+
+  const startColor = destructColor.apply({}, output[0]);
```

Comment:
is this called properly? Shouldn't it be `const startColor = destructColor(output[0]);`? `destructColor` (if I understand it correctly) is a function more or less like `(number) => Array<number>` so why calling `apply on it`?